examples/VMware/README.md

```diff
@@ -0,0 +1,21 @@
+# Integration of vSphere with Infoblox Provider

```

Comment:
Changes related to VMware is already there
why it is coming once again as new changes. Please rebase and submit once again

Reply:
you wanted to have to different tf files rather than one .That is why VMware has come